### PR TITLE
update to latest messaging

### PR DESCRIPTION
**Pull request by KPostOffice**
Use confluent kafka producer for messages. No longer require async tasks.

This should not affect performance because `confluent-kafka` wraps `librdkafka` which batches messages and sends them asynchronously.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,15 +4,16 @@ verify_ssl = true
...
 pyyaml = "*"
 thoth-storages = "*"
 thoth-common = "*"
 thoth-python = "*"
-thoth-messaging = "==0.7.13"
 jsonpath-ng = "*"
 prometheus-client = "*"
+thoth-messaging = "*"
+click = "*"
+typing-extensions = "*"
```
Comment (fridex) [inline Pipfile:16]: I don't see this package as a direct dependency in sources. If Pipenv is causing trouble with the installation process, you need to remove the virtual environment+Pipfile.lock and let Pipenv relock the dependency tree once again in a fresh environment.

**Review by fridex — COMMENTED** [review]
One minor comment, LGTM otherwise :+1:

**Review by pacospace — APPROVED** [review]
lgtm! +1

**Pull request merged**